index.ts

```diff
@@ -0,0 +1,26 @@
+import { AppSource } from "./util/database"
+import express from "express";
```

Comment:
It is a good practise to import the external npm packages first at the beginning of the file. It doesn't change anything, but it's better at least for me for some unknown reason.